examples/00-fluent/modeling_cavitation.py

```diff
@@ -0,0 +1,377 @@
+# Read and Check the Mesh
+# ~~~~~~~~~~~~~~~~~~~~~~~
+solver.read_case("cav.msh")
+
+solver.mesh.check()
+
+###############################################################################
+# Set units for length
+# ~~~~~~~~~~~~~~~~~~~~
+
+solver.tui.define.units("length", "m")
+
+###############################################################################
+# Specify an axisymmetric model
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+solver.setup.general.solver.two_dim_space = "axisymmetric"
+
+###############################################################################
+# Enable the multiphase mixture model
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+solver.setup.models.multiphase.models = "mixture"
+solver.tui.define.models.multiphase.mixture_parameters("no", "implicit")
+
+###############################################################################
+# Select turbulence model
+# ~~~~~~~~~~~~~~~~~~~~~~~
+
+# model : k-omega
+# k-omega model : sst
+
+solver.setup.models.viscous = {"model": "k-omega", "k_omega_model": "sst"}
+
+###############################################################################
+# Define materials
+# ~~~~~~~~~~~~~~~~
+# Create the material ``water``.
+
+# name: water
+# density : constant
+# reference density : 1000 [kg/m^3]
+# viscosity : constant
+# viscosity method : constant
+# reference viscosity : 0.001 [kg/(m s)]
+
+water = {
+    "density": {"option": "constant", "value": 1000},
+    "viscosity": {"option": "constant", "value": 0.001},
+}
+
+solver.setup.materials.fluid["water"] = water
+
+# Copy water vapor from the materials database and modify the properties of your local copy.
+
+solver.setup.materials.database.copy_by_name(type="fluid", name="water-vapor")
+
+# Set 0.02558 kg/m3 for Density and 1.26e-06 kg/m–s for Viscosity.
+
+water_vapor = {"density": {"value": 0.02558}, "viscosity": {"value": 1.26e-06}}
+
+solver.setup.materials.fluid["water-vapor"] = water_vapor
+
+###############################################################################
+# Phases
+# ~~~~~~
+# Change phase names to "liquid" and "water-vapor".
+
+solver.tui.define.phases.set_domain_properties.change_phases_names("vapor", "liquid")
+
+# Specify liquid water as the primary phase.
+
+solver.tui.define.phases.set_domain_properties.phase_domains.liquid.material(
+    "yes", "water"
+)
+
+# Specify water vapor as the secondary phase.
+
+solver.tui.define.phases.set_domain_properties.phase_domains.vapor.material(
+    "yes", "water-vapor"
+)
+
+# Enable the cavitation model
+
+# Set the number of mass transfer mechanisms to 1, set liquid as the from phase,
+# set vapor as the to phase and cavitation as the mechanism.
+
+solver.tui.define.phases.set_domain_properties.interaction_domain.heat_mass_reactions.mass_transfer(
+    1, "liquid", "vapor", "cavitation", "1", "no", "no", "no"
+)
+"""Set the vaporization pressure to 3540 Pa and bubble number density to 1e+11."""
```

Comment:
What is this here for?

Reply:
This was a command I could not find. These are default values which will not need to be set. This line has been removed.